db/schema.rb

```diff
@@ -10,11 +10,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_09_29_123828) do
+ActiveRecord::Schema.define(version: 2018_10_11_143410) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "addresses", force: :cascade do |t|
+    t.string "address_line_one", null: false
```

Comment:
⚠️ Seems like the schema is not up to date with the migration. Ensure the newly generated schema.rb is also pushed to the remote.

Reply:
pushed changed schema